### DISCUSSION TITLE
Error 404 while deploying my Vercel app

### DISCUSSION THREAD

**Discussion by ThiagoMarvinSoares** [vercel/community#6394] [Help]
### Summary

I Create a web app to generate a invoice to a company I work for, but when I try to generate it, give me back a 404 error.
My form is pointing to the right places and I don't understand why I'm getting this error.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/110858853/66870c15-f837-4f04-80b7-ebe4bf8c7e98)
![image](https://github.com/vercel/community/assets/110858853/d884b57c-2cb1-4794-9567-0bcadab99d9d)
![image](https://github.com/vercel/community/assets/110858853/7a61afa8-c75c-447e-9088-c4df0bf6b72e)

**Comment by versecafe** [comment]
@ThiagoMarvinSoares have you tested with something like Postman if the URL you're targeting exists and is actually running?

**Comment by ThiagoMarvinSoares** [comment]
I really don't know how to use it

**Comment by ThiagoMarvinSoares** [comment]
Actually I used it and get not_found there, strange is testing local it
works fine
